test/test-metadata.rb

```diff
@@ -0,0 +1,36 @@
+class MetadataTest < Test::Unit::TestCase
```

Comment:
Nothing below this line has been changed in this file.

Reply:
We can remove this file.